test/gateway/gateway.settleOrder.test.js

```diff
@@ -263,8 +268,8 @@ describe("Gateway settle order", function () {
 			.withArgs(
 				this.sender.address,
 				mockUSDT.address,
-				BigNumber.from(this.orderAmount).sub(this.protocolFee),
-				this.protocolFee,
+				this.orderAmount,
+				0,
```

Comment:
should be 27000

given your test

Reply:
instead of hardcoding, i calculated it, so kindly check